tests/v3_api/common.py

```diff
@@ -449,6 +449,21 @@ def validate_cluster(client, cluster, intermediate_state="provisioning",
                             rules=[rule])
     if not skipIngresscheck:
         validate_ingress(p_client, cluster, [workload], host, path)
+
+    if gkeIngresscheck:
+        name = random_test_name("testingress")
+        host = "xip.io"
+        path = "/name.html"
+        rule = {"host": host,
+                "paths":
+                    {path:
+                        {"workloadIds": [workload.id], "targetPort": "80"}}}
+        ingress = p_client.create_ingress(name=name,
+                                          namespaceId=ns.id,
+                                          rules=[rule])
+        print ingress
+        validate_gke_ingress(p_client, cluster, workload, ingress, path)
+
```

Comment:
@soumyalj We should be able to use the skipIngresscheck flag along with cluster information instead of a gkeIngresscheck flag .

Reply:
Removed the gkeingresscheck flag